script.js

```diff
@@ -1,12 +1,298 @@
-//You can edit ALL of the code here
+//Global Variables
+const episodeSearch = document.getElementsByClassName("season-names");
+const episodeCard = document.getElementsByClassName("card");
+const showCard = document.getElementsByClassName("show-card")
+const rootElem = document.getElementById("root");
+
+let inputSearch = document.createElement("INPUT");
+inputSearch.setAttribute("type", "search");
+
+let selector = document.createElement("select");
+selector.setAttribute("name", "episodes")
+selector.className = "selector";
+
+let shows = document.createElement("select");
+shows.setAttribute("name", "series")
+shows.className = "shows";
+
+let bod = document.getElementsByTagName("BODY")[0];
+let main = document.createElement("div")
+main.setAttribute("class", "main")
+bod.appendChild(main)
+
+let pages = document.createElement("div")
+pages.setAttribute("class", "pages")
+bod.appendChild(pages)
+
+
+let showId;
+let episode;
+
+//Setup Function
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  searchBar()
+  searchFilter()
+  showsList()
+  showSelection()
+  getShows()
+  citation()
+}
+
+//Populates the page with the shows
+function getShows(){
+  const rootElem = document.getElementById("root");
+
+  let shows = getAllShows();
+  let allShows = shows.sort(function compare(a, b) {
+    if (a.name.toUpperCase() < b.name.toLocaleUpperCase()) { return -1 }
+    if (a.name.toUpperCase() > b.name.toUpperCase()) { return 1 }
+    return 0;
+  })
+
+  for(i=0; i<allShows.length; i++){
```

Comment:
I've always preferred

```
allShows.forEach(show => { /* do stuff */ })
```

rather than the old school 'for loop'. not saying this is wrong, I just find it easier for my brain to process